index.html

```diff
@@ -50,7 +47,263 @@ <h6>LET'S CONNECT</h6>
+        </div>
+ 
+        <div class="small-buttons">
+            <ul>
+                <li> html</li> 
+                 <li>css</li> 
+                 <li>javascript</li>
+              </ul> 
+            </div>
+ 
+        <div class="big-button">
+                <button>
+                    See Project
+                </button>
+            </div>
+ 
+     </section>
+
+</section> <!--End of Wrapper-->
+
+  <section class="aboutme">
+ 
+    <div class="mytitle">
+        <h3>About <br> Myself</h3>
+    </div>
+
+    <div class="describe-me">
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="connect">
+          <h4>LET'S CONNECT</h4>
+          <ul>
+            <li><a href="https://twitter.com/"><img src="twitter.svg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/"><img src="linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="//www.medium.com/"><img src="m.svg" alt="medium"></a></li>
+            <li><a href="https://github.com/"><img src="github.svg" alt="github"></a></li>
+            <li><a href="peace"><img src="peace.svg" alt="peace"></a></li>
+        </ul>
+     </div>
+
+    <div class="big-button">
+                <button>
+                    Get my Resume
+                </button>
+            </div>  
+
+   <div class="lang">
+      <div class="text">
+          <p>
+              Languages
+          </p>
+        </div>
+      <div class="dropdown">
+           <select name="dropdown" id="dropdown">      
+           </select>
+        </div>
+    </div> 
+
+      <div class="programs">
+           <ul> 
+              <li class="java">
+                <img src="JS.png" alt="">
+                <p>javaScript</p>
+              </li>
+
+              <li class="java">
+                <img src="html.png" alt="">
+                <p>HTML</p>
+              </li>
+
+              <li class="java">
+                <img src="css.png" alt="">
+                <p>CSS</p>
+              </li>
+           </ul>
+      </div>
+           <div class="skills">
+            <div class="text">
+                <p>
+                    Frameworks
+                </p>
+            </div>
+            <div class="dropdown">
+                 <select name="dropdown">      
+                 </select>
+                </div> 
+            </div>
+<hr>
+                <div class="skills">
+                    <div class="text">
+                        <p>
+                            Skills
+                        </p>
+                    </div>
+                    <div class="dropdown">
+                         <select name="dropdown">      
+                         </select>
+                        </div> 
+                    </div>
+<hr> 
```

Comment:
- [ ] Kindly make sure you follow the design provided to meet the requirements.

- [ ] Your design 
![image](https://user-images.githubusercontent.com/55635977/164890240-f1873cec-76de-4149-9411-b366224ba283.png)

- [ ] The design provided
![image](https://user-images.githubusercontent.com/55635977/164890315-8e7b7286-5dfe-4a8b-bd02-e47964211139.png)

You can check the padding, the arrow direction on the design and font family.